contents/docs/product-analytics/autocapture.mdx

```diff
@@ -23,6 +23,182 @@ Autocapture is available in the following SDKs:
+<Tab.List>
+<Tab>Web</Tab>
+<Tab>iOS</Tab>
+<Tab>React Native</Tab>
+</Tab.List>
+<Tab.Panels>
+<Tab.Panel>
+
+| Event | Description |
+|-------|-------------|
+| `$autocapture` | On interactions with: `a`, `button`, `form`, `input`, `select`, `textarea`, `label`, and elements with `contenteditable="true"`. Autocaptures are [displayed](https://app.posthog.com/activity/explore) with names like `clicked span with text "Delete"`, but filtering for **Autocapture** will show you all autocaptured events. |
+| `$pageview` | When the user navigates to a new page. |
+| `$pageleave` | When the user navigates away from a page. |
+| `$rageclick` | A second click within 1000ms within 30px of the first click. |
+| `$copy_autocapture` | Clipboard events. Disabled by default. See [clipboard autocapture](#clipboard-autocapture). |
+
+Autocaptured events (and client-side custom events) have many default properties. These are distinguished by `$` prefix in their name, the PostHog logo next to them in the activity tab, and the verified event logo. You can find them in [PostHog](https://app.posthog.com/data-management/properties) or in [the references](https://posthog.com/docs/references/posthog-js/types/Properties).
+
+<details> 
+<summary>List of relevant default autocaptured properties</summary>
+
+| Name                        | Key                      | Example value                  |
+|-----------------------------|--------------------------|--------------------------------|
+| Timestamp                   | `$timestamp`             | `2024-05-29T17:32:07.202Z`     |
+| OS                          | `$os`                    | `Mac OS X`                     |
+| OS Version                  | `$os_version`            | `10.15.7`                      |
+| Browser                     | `$browser`               | `Chrome`                       |
+| Browser Version             | `$browser_version`       | `125`                          |
+| Device Type                 | `$device_type`           | `Desktop`                      |
+| Current URL                 | `$current_url`           | `https://example.com/page`     |
+| Host                        | `$host`                  | `example.com`                  |
+| Path Name                   | `$pathname`              | `/page`                        |
+| Screen Height               | `$screen_height`         | `1080`                         |
+| Screen Width                | `$screen_width`          | `1920`                         |
+| Viewport Height             | `$viewport_height`       | `950`                          |
+| Viewport Width              | `$viewport_width`        | `1903`                         |
+| Library                     | `$lib`                   | `web`                          |
+| Library Version             | `$lib_version`           | `1.31.0`                       |
+| Search Engine               | `$search_engine`         | `google`                       |
```

Comment:
I've never seen this one, since when do we have this?

Reply:
Perfect. Yeah I need to double check these with the code. there's stuff here not in my spec ;)